Don't set Helvetica as default font, it's not available as TrueType
font. Fixes Bug#477688.

 -- Rafael Laboissiere <rafael@debian.org>  Sat, 19 Jul 2008 10:27:26 +0200

diff -urNad octave3.0-3.0.1~/scripts/plot/__go_draw_axes__.m octave3.0-3.0.1/scripts/plot/__go_draw_axes__.m
--- octave3.0-3.0.1~/scripts/plot/__go_draw_axes__.m	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/scripts/plot/__go_draw_axes__.m	2008-05-25 17:30:08.000000000 +0200
@@ -1619,7 +1619,7 @@
 
 function [f, s, fnt, it, bld] = get_fontname_and_size (t)
   if (isempty (t.fontname))
-    fnt = "Helvetica";
+    fnt = "*";
   else
     fnt = t.fontname;
   endif
@@ -1657,7 +1657,7 @@
   if (strcmp (fld, "string"))
     [f, s, fnt, it, bld] = get_fontname_and_size (obj);
   else
-    f = "Helvetica";
+    f = "*";
     s = 10;
     fnt = f;
     it = false;
